roles/vmrole/tasks/generatevm.yml

```diff
@@ -2,10 +2,27 @@
 ---
 # TODO: fetch template from API once loaded at deployment
 # TODO: Proper Linux and Windows templates are not yet ready, using the latest available
-- name: Fetch template
+
+- name: Set PVC_NAME if not set 
+  set_fact:
+    PVC_NAME: windows-vm-pvc-{{ NAME }}
+  when: PVC_NAME == ''
```

Comment:
If you set ```PVC_NAME``` does that mean you having an existing pvc?

Reply:
no, if you dont set a PVC_NAME, it s that you rely on the apb to create one for you

Reply:
What I'm trying to ask is if ```PVC_NAME != ''```will ```existing_pvc``` always be true?